maps/tether/tether_jobs.dm

```diff
@@ -138,11 +138,14 @@
 	timeoff_factor = 1
 	access = list(access_talon)
 	minimal_access = list(access_talon)
-	alt_titles = list("Talon Security" = /datum/alt_title/talon_security)
+	alt_titles = list("Talon Security" = /datum/alt_title/talon_security, "Talon Marine" = /datum/alt_title/talon_marine)
 
 /datum/alt_title/talon_security
 	title = "Talon Security"
 
+/datum/alt_title/talon_marine
```

Comment:
as I said on the other thing, I really do not want 'Talon Marine' to be a thing, as it evokes too much 'military' 'soldier' vibes, and that's not what the job is or should be.

Reply:
I'll change it up when I get home!